app/api/fetch-email-from-source.jsx

```
@@ -2,18 +2,20 @@ import axios from "axios";
   try {
     const prisma = new PrismaClient();
-    const { sourceId, days = 7 } = await request.json();
-
-    if (!sourceId) {
-      return new Response(JSON.stringify({ error: "SourceId is required" }), {
-        status: 400,
-        headers: { "Content-Type": "application/json" },
-      });
+    const { days = 7 } = await request.json();
+    const sourceId = parseInt(params.sourceId, 10);
+    if (isNaN(sourceId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)